connector_search_engine/models/se_backend.py

```diff
@@ -96,31 +81,17 @@ def _compute_specific_backend(self):
                     spec_backend._name, spec_backend.id
                 )
 
-    @api.onchange("name")
-    def _onchange_name(self):
-        if self.name and not self.tech_name:
-            self.tech_name = self.name
-
     @api.onchange("tech_name", "index_prefix_name")
     def _onchange_tech_name(self):
-        if self.tech_name:
-            # make sure is normalized
-            self.tech_name = self._normalize_name(self.tech_name)
+        super()._onchange_tech_name()
         if self.index_prefix_name:
             # make sure is normalized
-            self.index_prefix_name = self._normalize_name(self.index_prefix_name)
+            self.index_prefix_name = self._normalize_tech_name(self.index_prefix_name)
         else:
```

Comment:
```suggestion
        elif self.tech_name:
```
as tech_name is not required anymore

Reply:
it is https://github.com/OCA/server-env/pull/52/files#diff-9fb128f0d1213751a2f5f5c5f3c3c382R36